cmd/zoekt-sourcegraph-indexserver/index.go

```diff
@@ -270,7 +271,12 @@ func gitIndex(c gitIndexConfig, o *indexArgs) error {
 		}
 	}
 
-	debug.Printf("successfully fetched git data for %q (%d commit(s)) in %s", o.Name, successfullyFetchedCommitsCount, fetchDuration)
+	logger.Debug("successfully fetched git data",
+		zap.String("repo", o.Name),
```

Comment:
We encourage the use of `sglog.String` etc for all fields, because we hook into its behaviour (for example, [errors](https://sourcegraph.com/github.com/sourcegraph/log/-/blob/fields.go?L72-83) and depend on these customizations. There are also explorations to extend behaviours further at the field level (https://github.com/sourcegraph/sourcegraph/issues/33473).

In `sourcegraph/sourcegraph` we have a linter that forbids all direct imports of `zap`, I recommend something similar be set up here to avoid unexpected divergences

Reply:
fixed